conda-store-server/conda_store_server/build.py

```diff
@@ -10,29 +10,56 @@
 
 import yaml
 
+from filelock import FileLock
 from sqlalchemy.orm import Session
 
 from conda_store_server import action, api, conda_utils, orm, schema, utils
 from conda_store_server.utils import BuildPathError
 
 
-def append_to_logs(db: Session, conda_store, build, logs: typing.Union[str, bytes]):
-    try:
-        current_logs = conda_store.storage.get(build.log_key)
-    except Exception:
-        current_logs = b""
+class LoggedStream:
+    """Allows writing to storage via logging.StreamHandler"""
```

Comment:
This class is designed to be used with `StreamHandler`. From https://docs.python.org/3/library/logging.handlers.html#streamhandler:

> The StreamHandler class, located in the core logging package, sends logging output to streams such as sys.stdout, sys.stderr or any file-like object (or, more precisely, any object which supports write() and flush() methods).

Reply:
Can you leave this useful information in the docstring?